MantleTests/MTLTestModel.m

```diff
@@ -472,3 +472,37 @@ - (NSString *)flavor {
 @implementation MTLOptionalPropertyModel
 
 @end
+
+@implementation MTLRecursiveUserModel
+
++ (NSDictionary *)JSONKeyPathsByPropertyKey {
+	return @{
+		@"name": @"name",
+		@"groups": @"groups",
+	};
+}
+
++ (NSValueTransformer *)groupsJSONTransformer {
+	return [MTLJSONAdapter arrayTransformerWithModelClass:[MTLRecursiveGroupModel class]];
+}
+
+@end
+
+@implementation MTLRecursiveGroupModel
+
++ (NSDictionary *)JSONKeyPathsByPropertyKey {
+	return @{
+		@"owner": @"owner",
+		@"users": @"users",
+	};
+}
+
++ (NSValueTransformer *)ownerJSONTransformer {
+	return [MTLJSONAdapter dictionaryTransformerWithModelClass:[MTLRecursiveUserModel class]];
+}
+
++ (NSValueTransformer *)usersJSONTransformer {
+	return [MTLJSONAdapter arrayTransformerWithModelClass:[MTLRecursiveUserModel class]];
```

Comment:
Can you use dot notation here and in the other calls to `class` below as well? Sorry for not pointing this out earlier :sweat: